lib/Runtime/Library/SIMDUint8x16Lib.cpp

```diff
@@ -265,7 +264,7 @@ namespace Js
 
         if (args.Info.Count >= 4 && JavascriptSIMDUint8x16::Is(args[3]))
         {
-            SIMD128TypedArrayStore<JavascriptSIMDUint8x16>(args[1], args[2], args[3], 16 * TySize[TyUint8], scriptContext);
+            SIMD128TypedArrayStore<JavascriptSIMDUint8x16>(args[1], args[2], args[3], 16 * INT8_SIZE, scriptContext);
```

Comment:
Simd types describe the size of a lane first (number of bits) followed by the number of lanes. So Uint8x16 means that the SIMD vector has 16 lanes of 8bit uints. This is true for all SIMD types. This is according to the SIMD specification.